### PR TITLE
Misc fixes

### PR DESCRIPTION
**Pull request by westonruter**

### CODE REVIEW

File: init-plugin.sh [hunk 1]
```diff
@@ -39,14 +37,18 @@ echo "Prefix: $prefix"
 echo "NS: $namespace"
 echo "Class: $class"
 
-git clone --recursive https://github.com/xwp/wp-foo-bar.git "$slug"
+git clone "$(dirname "$0")" "$slug"
```
Comment (westonruter) [inline init-plugin.sh]: This change will be helpful as you can create a new plugin without an internet connection. So you can do:

```
cd wp-content/plugins
./foo-bar/init-plugin.sh "My Awesome Plugin"
```

File: init-plugin.sh [hunk 2]
```diff
@@ -26,8 +26,6 @@ prefix=$( perl -pe '$_ = lc; s/ /_/g' <<< "$name" )
 namespace=$( perl -pe 's/ //g' <<< "$name" )
 class=$( perl -pe 's/ /_/g' <<< "$name" )
 
-cd ..
```
Comment (valendesigns) [inline init-plugin.sh]: Why remove this and create the plugin inside the `foo-bar` directory? Is there a benefit or limitation that I'm not aware of? It's so much nicer to be able to activate the plugin immediately without having to perform any other commands or move the directory manually.

Comment (valendesigns) [inline init-plugin.sh]: I guess if you're doing it the way you mentioned above like:
```
cd wp-content/plugins
./foo-bar/init-plugin.sh "My Awesome Plugin"
```

However, I was doing it like the direction in the readme like so
```
cd wp-content/plugins/foo-bar
./init-plugin.sh "My Awesome Plugin"
```

If you do it this way the directory is created inside the `foo-bar` directory which is kind of weird. I suggest we do a check for the current working directory and if it's `foo-bar` then we `cd ..` back up to the plugins directory to create the new plugin. That way both use cases are handled and it does create plugins inside plugins. :smile:

Comment (westonruter) [inline init-plugin.sh]: Good call. Done in e2c1239

**Pull request merged**